README.md

```diff
@@ -1,19 +1,25 @@
-# Ubuntu Touch
+------------------------------------------------------------------------
 
-## A simple and beautiful mobile OS for everyone!
+# <a href="https://imgbb.com/"><img src="https://i.ibb.co/5WgVdC1/uports.png" alt="uports" border="0"></a> &nbsp; Ubuntu Touch   &nbsp; <a href="https://imgbb.com/"><img src="https://i.ibb.co/5WgVdC1/uports.png" alt="uports" border="0"></a>
```

Comment:
Hotlinking to uncontrolled image sites generally isn't a very good idea... Since this is GitHub, you can add this image to the repository and use it in the same way as other image.

Reply:
Alright gotcha! I will add a change to it soon as I'm free. :)
(I'll add a file for it called 'repo-images')